Create GitHub Pages documentation site with Jekyll (#329)
moov-io/watchman: docs/ha.md

---
layout: page
title: High Availability
hide_hero: true
show_sidebar: false
menubar: docs-menu
---

# High Availability

> Watchman doesn't currently support operating in a high availability (HA) mode. The major reason of doing so would be to distribute webhook notifications for high volume watches. We've not concerned ourselves with this as the notification rate is higher than anything we've needed at Moov. The complexity of implementing HA is estimated to be too much for our current needs.

Reviewer: ```suggestion
Watchman doesn't currently support operating in a high availability (HA) mode. The major reason of doing so would be to distribute webhook notifications for high volume watches. The complexity of implementing HA is estimated to be too much for the current needs of this project.
```
